examples/basic/main.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/paked/messenger"
 )
 
+//profileField is a slice of strings of the user profile field the developer wants access
+var (
+	profileField = []string{"name", "first_name", "last_name", "profile_pic"}
```

Comment:
I would inline this slice, so it looks like:

```
p, err := client.ProfileByID(m.Sender.ID, []string{"name", "first_name", "last_name", "profile_pic"})
```

It's only used in one place, so we don't need to add an extra level of indirection.

Reply:
+1